mifosng-android/src/main/java/com/mifos/api/services/DocumentService.java

```diff
@@ -74,10 +74,10 @@
      *                                or Savings etc)
      * @param entityId      - Id of Entity
      * @param documentId    - Document Id
-     * @return
+     * @return DocumentRelatedResponse
      */
     @DELETE("{entityType}/{entityId}/" + APIEndPoint.DOCUMENTS + "/{documentId}")
-    Observable<GenericResponse> removeDocument(@Path("entityType") String entityType,
+    Observable<DocumentRelatedResponse> removeDocument(@Path("entityType") String entityType,
```

Comment:
@tarun0 When we know what will be response for many requests the It's better to follow genericity and Please use `GenericResponse`